packages/castmill/lib/castmill/addons/playlists/components/index.tsx

```diff
@@ -30,61 +28,38 @@ const PlaylistsPage: Component<{
   store: AddonStore;
   params: any; //typeof useSearchParams;
 }> = (props) => {
-  const toast = useToast();
   const [data, setData] = createSignal<JsonPlaylist[]>([]);
   const [currentPlaylist, setCurrentPlaylist] = createSignal<JsonPlaylist>();
   const [showModal, setShowModal] = createSignal(false);
 
-  const [showAddPlaylistModal, setShowAddPlaylistModal] = createSignal(false);
-  const [selectedPlaylists, setSelectedPlaylists] = createSignal(
-    new Set<number>()
-  );
-
-  const [quota, setQuota] = createSignal<ResourceQuota | null>(null);
-  const [quotaLoading, setQuotaLoading] = createSignal(false);
-  const [showLoadingIndicator, setShowLoadingIndicator] = createSignal(false);
-
-  const quotasService = new QuotasService(props.store.env.baseUrl);
-
-  let loadingTimeout: ReturnType<typeof setTimeout> | undefined;
-
-  const loadQuota = async () => {
-    if (!props.store.organizations.selectedId) return;
-    
-    try {
-      setQuotaLoading(true);
-      
-      // Only show loading indicator if request takes more than 1 second
-      loadingTimeout = setTimeout(() => {
-        if (quotaLoading()) {
-          setShowLoadingIndicator(true);
+  const [teams, setTeams] = createSignal<Team[]>([]);
+  const [selectedTeamId, setSelectedTeamId] = createSignal<number | null>(null);
+
+  // Fetch teams for the organization
+  createEffect(async () => {
```

Comment:
Same as above. In fact, you should probably be able to also create a separate component for this and reuse it everywhere.